vega/base.py

```diff
@@ -51,6 +53,8 @@ def _ipython_display_(self):
             metadata={'jupyter-vega': '#{0}'.format(id)}
         )
 
-    def display(self):
+    def display(self, renderer=None):
         """Render the visualization."""
+        if renderer:
```

Comment:
Why is this necessary? Isn't `self.renderer` not defined the same as setting it to `None`?